dev/tasks/python-wheels/travis.osx.yml

```diff
@@ -31,7 +31,7 @@ addons:
       - git
       - openssl@1.1
       - protobuf
-      - python@3.8
+      - python@3.9
       - thrift
       - wget
     update: false
```

Comment:
I think it's possible Travis doesn't have Python 3.9 in it's homebrew cache, but I would expect a different error than it succeeds and then `/usr/local/opt/python@3.8/libexec/bin/pip` is just not found...

Nevermind! Apparently it's just a "warning" :unamused: :
> Warning: 'python@3.9' formula is unreadable: No available formula with the name "python@3.9" 

https://travis-ci.org/github/ursa-labs/crossbow/builds/733873746#L145